src/components/Subthought.tsx

```diff
@@ -114,6 +116,34 @@ const Subthought = ({
     ref.current.style.opacity = opacity
   })
 
+  /**
+   * Cleans up editable classes from the provided HTML string.
+   *
+   * @param htmlString - The HTML string to clean up.
+   * @returns The cleaned HTML string.
+   */
+  const cleanUpEditableClasses = (htmlString: string) => {
+    const container = document.createElement('div')
+    container.innerHTML = htmlString
+    const editableElements = container.querySelectorAll('.editable--preventAutoscroll_true')
```

Comment:
`.editable--preventAutoscroll_true` is autogenerated by PandaCSS so it's dangerous to hardcode it into the component. If the [recipe variant](https://github.com/cybersemics/em/blob/42b1ef55abd5457ecbbbc39512be3dc6a1682fb5/src/recipes/editable.ts#L41) changes, this will break without us knowing.

I believe PandaCSS has a way to programmatically get the className of a recipe variant, although I don't know the technique off the top of my head. Could you use that here? If you can't find it let me know.